packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -185,12 +241,25 @@ def imshow(
         )
 
     layout_patch = dict()
-    for v in ["title", "height", "width"]:
-        if args[v]:
-            layout_patch[v] = args[v]
+    for attr_name in ["title", "height", "width"]:
+        if args[attr_name]:
+            layout_patch[attr_name] = args[attr_name]
     if "title" not in layout_patch and args["template"].layout.margin.t is None:
         layout_patch["margin"] = {"t": 60}
     fig = go.Figure(data=trace, layout=layout)
     fig.update_layout(layout_patch)
+    if img_is_xarray:
```

Comment:
I actually think we can do this for `np` inputs also, no, given the `labels` array? We can just default `x_label` to `x` etc